<ui version="4.0" >
 <class>FilterDialog</class>
 <widget class="QDialog" name="FilterDialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>375</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>375</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Choose filter type</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="horizontalLayout" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>260</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton" >
      <property name="text" >
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okButton" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayout" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>10</y>
     <width>111</width>
     <height>235</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QLabel" name="label" >
      <property name="font" >
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>Select filter</string>
      </property>
      <property name="buddy" >
       <cstring>comboBox</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>1</hsizetype>
        <vsizetype>0</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editable" >
       <bool>false</bool>
      </property>
      <property name="sizeAdjustPolicy" >
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="duplicatesEnabled" >
       <bool>true</bool>
      </property>
      <property name="frame" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox" >
      <property name="text" >
       <string>custom</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Rows</string>
        </property>
        <property name="buddy" >
         <cstring>spinBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>4</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum" >
         <number>15</number>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="singleStep" >
         <number>2</number>
        </property>
        <property name="value" >
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Cols</string>
        </property>
        <property name="buddy" >
         <cstring>spinBox_2</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_2" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>4</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum" >
         <number>15</number>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="singleStep" >
         <number>2</number>
        </property>
        <property name="value" >
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_2" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>symetric</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>Save mask</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteButton" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>Delete mask</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>241</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>0</hsizetype>
     <vsizetype>0</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="verticalScrollBarPolicy" >
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy" >
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="editTriggers" >
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors" >
    <bool>false</bool>
   </property>
   <property name="selectionMode" >
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="textElideMode" >
    <enum>Qt::ElideMiddle</enum>
   </property>
   <property name="rowCount" >
    <number>3</number>
   </property>
   <property name="columnCount" >
    <number>3</number>
   </property>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>340</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel" >
     <x>338</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>348</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel" >
     <x>355</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>checkBox_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>353</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel" >
     <x>354</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>FilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>288</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>FilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>okButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>338</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel" >
     <x>332</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>saveButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>366</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel" >
     <x>273</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>deleteButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>286</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel" >
     <x>288</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
